src/main/kotlin/lotto/domain/WinningNumbers.kt

```diff
@@ -0,0 +1,22 @@
+package lotto.domain
+
+import lotto.constant.LottoConstants
+
+class WinningNumbers(private val numbers: List<LottoNumber>) {
+
+    init {
+        require(numbers.size == LottoConstants.NUMBERS_AMOUNT) {
+            "There must be ${LottoConstants.NUMBERS_AMOUNT} winning numbers."
+        }
+    }
+
+    companion object {
+        fun ofNumbers(numbers: List<Int>): WinningNumbers {
+            numbers.map { LottoNumber(it) }
+                .toList()
+                .also { return WinningNumbers(it) }
+        }
```

Comment:
1. I think map already return the list so `toList` maybe doesn't need
2. `also` is usually used in debugging, logging. So I think returning something in also is liitle bit weird
```suggestion
        fun ofNumbers(numbers: List<Int>): WinningNumbers =
    WinningNumbers(numbers.map { LottoNumber(it) })
```

Reply:
you're the best